### DISCUSSION TITLE
Port Martin Zuther's compressor Squeezer.

### DISCUSSION THREAD

**Discussion by 38github** [DISTRHO/DISTRHO-Ports#122] [Ideas]
I would love to see this become a DISTRHO-port. https://github.com/mzuther/Squeezer

**Comment by falkTX** [comment]
why is that needed? plugin is open and works on linux already

**Reply by 38github** [reply]
It is not available as LV2 any longer if I am correct.

**Reply by falkTX** [reply]
correct. but since LV2 is now officially supported in JUCE it is mostly a matter of updating the plugin framework and enabling a few options.
ask the developer about it first, then come back if there is no wish at all for an lv2 version on their side.
